cmd/kubeapps-apis/proto/kubeappsapis/apidocs/v1/apidocs.proto

```diff
@@ -1,5 +1,5 @@
 syntax = "proto3";
-package kubeappsapis.apidocs;
+package kubeappsapis.apidocs.v1;
```

Comment:
We should similarly be using v1alpha1 here so we don't find ourselves caught out later (which means moving the directory).

Reply:
Ah, that's totally true... I just used the suggested one by the linter, but, you're right, better just use the same `v1alpha1`. Will update it shortly!

Reply:
Just note, I think you only renamed the package name to `v1alpha1`, so `buf lint` will probably complain that the file should be in the directory `cmd/kubeapps-apis/proto/kubeappsapis/apidocs/v1alpha1/apidocs.proto` . But we can just squeeze that in with another PR. In the future we can ensure that `buf lint` is required to exit 0 when building the image so will be caught by CI.

Reply:
>  In the future we can ensure that `buf lint` is required to exit 0 when building the image so will be caught by CI.

Oops, that's true. We certainly need adding at list a linter step in the CI, I've added the card for the next iteration discussion. Thanks for pointing this out!